ios/Plugin/Plugin/Rsp/RspError.swift

```diff
@@ -36,7 +38,7 @@ public class RspError: Rsp {
      
      - Returns: A dictionary representing the error data in a format suitable for a Capacitor plugin call result.
      */
-    override func toPluginCallResultData() -> [String: Any] {
+    override func toPluginCallResultData() -> PluginCallResultData {
```

Comment:
Is this type casting necessary? If it is, it is better if done in the plugin layer instead, to decouple this code from capacitor

Reply:
Resolved